core/src/main/java/com/onelogin/saml2/http/HttpRequest.java

```diff
@@ -10,15 +10,25 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.onelogin.saml2.util.Util;
 
 /**
  * Framework-agnostic representation of an HTTP request.
  *
  * @since 2.0.0
  */
 public final class HttpRequest {
+	
+	public static final Map<String, List<String>> EMPTY_PARAMETERS = Collections.<String, List<String>>emptyMap(); 
```

Comment:
Ah, whitespace. That was sloppy of me. I'm used to auto-format-on-save in my normal setup. I can commit a cleanup patch. Just fixing the added stuff, adapting to the formatting used in each file.